lib/geokit/net_adapter/typhoeus.rb

```diff
@@ -2,7 +2,8 @@ module Geokit
   module NetAdapter
     class Typhoeus
       def self.do_get(url)
-        ::Typhoeus.get(url.to_s)
+        Geokit::Geocoders.useragent ? headers = {'User-Agent' => Geokit::Geocoders.useragent} : headers = {}
```

Comment:
Can you address this comment, e.g. headers = Geokit::Geocoders.useragent ? {'User-Agent' => Geokit::Geocoders.useragent} : {}